validator/api/src/test/java/tech/pegasys/teku/validator/api/KeyStoreFilesLocatorTest.java

```diff
@@ -228,6 +250,29 @@ private void createFiles(final Path tempDir, Path... paths) throws IOException {
     }
   }
 
+  private void writeDepositDataFile(final Path depositDataFilePath) throws IOException {
+    String jsonData =
+        "[\n"
+            + "{\n"
+            + "  \"pubkey\": \"a1a203\",\n"
+            + "    \"withdrawal_credentials\": \"005058e2\",\n"
+            + "    \"amount\": 32000000000,\n"
+            + "    \"signature\": \"996413ecf1b6f07208f\",\n"
+            + "    \"deposit_message_root\": \"79d76d1\",\n"
+            + "    \"deposit_data_root\": \"f923110\",\n"
+            + "    \"fork_version\": \"00002009\",\n"
+            + "    \"eth2_network_name\": \"pyrmont\",\n"
+            + "    \"deposit_cli_version\": \"1.2.0\"\n"
+            + "}\n"
+            + "]\n";
+
+    byte[] data = jsonData.getBytes(UTF_8);
+    File file = depositDataFilePath.toFile();
+    if (file != null) {
```

Comment:
I'd omit this null check - if the file is null for some reason we want the test to fail anyway.